client/js/components/statement/assessmentTable/DpExportModal.vue

```diff
@@ -279,73 +253,59 @@
               <legend
                 class="sr-only"
                 v-text="Translator.trans('export.type')" />
-              <label
-                for="xlsxAnonymous"
-                class="u-mb-0_25">
```

Comment:
here and in several other cases: is the margin-bottom applied in <dp-checkbox />?

Reply:
No, it wasn't. :disappointed: I now took a look at the spacing and decided to only keep the margin-bottom on radio buttons that are not the last button in the fieldset, to ensure consistent spacing: 68ade91523383869edf47fd2d597bbfaa14467c4 (so margin-bottom is removed entirely from checkboxes, as there is only ever one)